nov-xwidget.el

```
@@ -34,6 +34,7 @@
 (require 'shr)
 (require 'xwidget)
 (require 'cl-lib)
+(require 'evil-core)
```

```suggestion
(require 'evil-core nil 'noerror)
```
User may not have Evil installed, ignore the error if not exists.